src/strings/index.tsx

```diff
@@ -646,6 +646,9 @@ const strings = new LocalizedStrings({
     TOTAL_SEEDS_TESTED: 'Total Seeds Tested',
     APPLY_RESULT_QUESTION: 'Do you want to apply this result to the accession?',
     APPLY_RESULT: 'Apply Result',
+    SEEDS_TESTED_ERROR: 'Seeds tested should be less than remaining seeds',
+    SEEDS_GERMINATED_ERROR: 'Germinated amount should be less than tested amount.',
```

Comment:
also might help to rename the key to `TOTAL_SEEDS_TESTED_ERROR` same for germinated